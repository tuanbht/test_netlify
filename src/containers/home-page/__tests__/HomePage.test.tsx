import { shallow } from 'enzyme';
import HomePage from '../index';
import React from 'react';

describe('HomePage', () => {
  it('renders template exactly', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
