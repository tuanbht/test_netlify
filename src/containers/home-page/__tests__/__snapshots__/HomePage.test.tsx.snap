// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders template exactly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-content-2"
  >
    <span
      className="makeStyles-title-3"
    >
      Coming soon...
    </span>
    <ForwardRef(SvgPalomaLogo)
      className="makeStyles-logo-4"
    />
  </div>
</div>
`;
